### DISCUSSION TITLE
What are the best hardware and hydrophone sources for Orcasound in 2024?

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/orcanode#68] [General]
Are you happy with your current hardware at the Orcasound hydrophone location you host? If so, what will it cost to purchase a replacement set up?

Here are links to recent hardware solutions: [2024 typical set up](https://github.com/orcasound/orcanode/wiki/Current-hardware-software-solution); [2018 blog post with some updates](https://www.orcasound.net/2018/04/27/orcasounds-new-live-audio-solution-from-hydrophone-to-headphone-with-a-raspberry-pi-computer-and-hls-dash-streaming-software/).

If you're not perfectly happy, can you find and share new sources of components that will work with Orcasound software and possibly reduce our collective expenses over time? Can we aggregate our purchasing to reduce our costs together?

Let's work together to keep getting recent cost estimates from manufacturers and distributors and sharing them in the [Hydrophone system comparison matrix](https://docs.google.com/spreadsheets/d/1yRxvllcRFqCD2XoBE96YamPgboD5sRBOzdUdtEHt2kk/edit?gid=0#gid=0)!

**Comment by scottveirs** [comment]
Latest discount from Adam at CRT:

"Yes, you are correct, 23% discount for those that purchase for the Orcasound network. Some common options, with pricing, are listed below. 

Pricing chart on the CRT-40P hydrophone:

1. 10m CRT-40P: $499 MSRP / $385 Orcasound discount price

2. 30m CRT-40P: $599 MSRP / $459 Orcasound discount price

3. 50m CRT-40P: $699 MSRP / $539 Orcasound discount price

4. 60m CRT-40P: $749 MSRP / $577 Orcasound discount price

5. Upgrade to a listening kit with all accessories (recorder, speaker, bluetooth transmitter, hardshell storage case, lanyard, and memory card): $300 MSRP additional / $231 Orcasound discount price"

**Comment by dbainj1** [comment]
The Reson TC-4032 has the specs I'd want in a research grade hydrophone, but is overkill for Orcasound.
